std/concurrency.d

```diff
@@ -198,7 +199,8 @@ private
 
                 foreach (t2; T[i + 1 .. $])
                 {
-                    static assert(isFunctionPointer!t2 || isDelegate!t2);
+                    static assert(isFunctionPointer!t2 || isDelegate!t2,
+                            "Not all passed T are function pointer or delegates");
```

Comment:
this static assert is redundant with the one above, no?

Reply:
done